docs/ECO-INIT.md

```diff
@@ -0,0 +1,153 @@
+# ECO Initialization
+
+This document describes the ECO initialization process, specifically how
+an ECO goes from "configured in the controller" to "launched and running".
+This includes:
+
+* configuration creation
+* image download and verification
+* network creation and initialization
+* ECO launch
+
+For a deeper understanding off ECOs, please read [ECOs.md](./ECOs.md).
+
+## Process
+
+The high-level process for preparing an ECO follows this flow. In each section, the
+responsible component is placed in brackets. All communication between components,
+i.e. configs and statuses, are performed using publish-subscribe via
+[pubsub](./IPC.md).
+
+1. Create App: An ECO is configured on a valid controller, e.g. [zedcloud](https://zededa.com)
+1. Receive Device Config ([zedagent](../pkg/pillar/cmd/zedagent)): EVE device receives and [processes](../pkg/pillar/cmd/zedagent/handleconfig.go#L412) an updated device config message which contains the new ECO app
+1. Create Local App Config ([zedagent](../pkg/pillar/cmd/zedagent)): [parse the app config](../pkg/pillar/cmd/zedagent/parseconfig.go#L453) in the device config message to [create and publish](../pkg/pillar/cmd/zedagent/parseconfig.go#L491-558) an [AppInstanceConfig](../pkg/pillar/types/zedmanagertypes.go#L46) for the ECO
+1. Consolidate ECO management ([zedmanager](../pkg/pillar/cmd/zedmanager/)): [Retrieve and parse](../pkg/pillar/cmd/zedmanager/zedmanager.go#L443-L%40) the [AppInstanceConfig](../pkg/pillar/types/zedmanagertypes.go#L46), and launch the ECO using the following steps.
+   1. Retrieve and verify image
+   1. Prepare ECO networking
+   1. Launch ECO
+
+`zedmanager` itself does not do any of the downloading, preparation or launching. It only _coordinates_ other
```

Comment:
I don't think we refer to what it does as coordinating elsewhere. But we don't have a zedmanager.md file yet!
How about 'orchestrate' or 'sequence' other microservices (not 'processes')?

Reply:
I am fine with changing it to "orchestrate", although I am unclear on how that is different from "coordinate".

Why not processes? They actually are processes, are they not?